docs/proposals/images/machine-states-preboot/Makefile

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2018 The Kubernetes authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+.PHONY: figures
+
+figures: *.png plantuml.jar
```

Comment:
we should really be doing this via containers IMO.

Reply:
I tried existing public containers. None of them rendered all the characters.

I know I can make it work with a different base image and the newest PlantUML version. Should I publish my own container and reference it here?

Reply:
Let's talk with google folks to see if we can get some build containers in place. 
/cc @BenTheElder @spiffxp

Reply:
i'm not sure where this image would be hosted, I'm still waiting on wg-k8s-infra to give arbitrary repos official hosting, keys etc. kind is still quietly using a docker-hub :upside_down_face:

Reply:
I'll need to figure out a base image with a font that renders the checkmarks.

https://forum.plantuml.net/6114/default-font-used

Reply:
paging - @dims

Reply:
I created my own image, based on [the Dockerfile of the official plantuml-server](https://github.com/plantuml/plantuml-server/blob/master/Dockerfile) image.

The checkmark (✅) codepoint did not render correctly because a font was missing. I found out which font(s) were capable of rendering this codepoint with [this helpful script](https://unix.stackexchange.com/questions/162305/find-the-best-font-for-rendering-a-codepoint/162334).

I installed fonts-symbola to the image, and the checkmark rendered correctly.

I am hosting the image on [Docker hub](https://hub.docker.com/r/dpf9/plantuml). The Dockerfile for my image is https://github.com/dlipovetsky/plantuml-docker.

Reply:
/uncc
I'm fine with the dockerhub container for now, if you want to host on gcr.io open a PR against kuberentes/k8s.io, ref: https://github.com/kubernetes/k8s.io/issues/158#issuecomment-505946374